modules/gltf/doc_classes/EditorSceneImporterGLTF.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EditorSceneImporterGLTF" inherits="EditorSceneImporter" version="3.4">
+	<brief_description>
```

Comment:
I think it's time to document gltf for master and 3.x. Starting with a master pr, so it be a separate pr.

Reply:
I think that's a separate issue from the code backport itself. The lack of documentation shouldn't block this PR.